feature-libs/product-configurator/rulebased/components/attribute/types/input-field/configurator-attribute-input-field.component.spec.ts

```diff
@@ -25,12 +29,15 @@ describe('ConfigAttributeInputFieldComponent', () => {
   beforeEach(
     waitForAsync(() => {
       TestBed.configureTestingModule({
-        declarations: [
-          ConfiguratorAttributeInputFieldComponent,
-          MockFocusDirective,
-        ],
+        declarations: [ConfiguratorAttributeInputFieldComponent],
         imports: [ReactiveFormsModule],
-        providers: [ConfiguratorAttributeBaseComponent],
+        providers: [
+          ConfiguratorAttributeBaseComponent,
```

Comment:
I see this BaseComponent in `providers` in many tests. It is not dependency.

Reply:
picked